src/shared/ui/survey/RadioActivityItem/RadioItem.tsx

```diff
@@ -83,20 +86,26 @@ const RadioItem: FC<RadioLabelProps> = ({
         <Box width="10%">
           <CachedImage
             resizeMode="contain"
+            data-test={`radio-option-image-${value}`}
```

Comment:
What is the value here? True / false or text ?

Reply:
value is number - unique for each option in radio group